shared/tracker/index.js

```diff
@@ -144,7 +93,18 @@ export default connect(
   }),
   dispatch => {
     return bindActionCreators(trackerActions, dispatch)
-  })(Tracker)
+  },
+  (stateProps, dispatchProps, ownProps) => ({
+    ...ownProps,
+    ...stateProps,
+    ...dispatchProps,
+    onClose: () => dispatchProps.onClose(ownProps.username),
```

Comment:
I don't understand this, could you please elaborate on what "merge function" is referring to? Object merge?